tsconfig.build.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "./tsconfig",
+    "exclude": [
+        "RNFBSDKExample"
+    ]
```

Comment:
Add `"RNFBSDKExample"` to `exclude` for typescript build otherwise it will build files end with `.ts`/.`tsx` in the example which is not expected.